src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Data/Migrations/20210902111722_Initial.cs

```diff
@@ -222,8 +229,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                         name: "FK_UserPreferences_AspNetUsers_ApplicationUserId",
                         column: x => x.ApplicationUserId,
                         principalTable: "AspNetUsers",
-                        principalColumn: "Id",
-                        onDelete: ReferentialAction.Restrict);
+                        principalColumn: "Id");
```

Comment:
I assume these changes are auto-generated (in which case great), otherwise I don't know why we'd be removing details like the `onDelete` here.

Reply:
Yep, just removed the migrations and recreated them